src/js/controllers/mapController.ts

```diff
@@ -63,11 +85,54 @@ export class MapController {
               title,
               opacity,
               visible,
-              definitionExpression
+              definitionExpression,
+              group: 'webmap'
             });
           });
+
           store.dispatch(allAvailableLayers(mapLayerObjects));
 
+          this.getMoreLayers().then(res => {
+            const { appState } = store.getState();
+
+            const resourceLayerObjects: LayerProps[] = [];
+
+            res.forEach((apiLayer: RemoteDataLayer) => {
```

Comment:
We need to add check if `apiLayer` exists like this: `if (!apiLayer) return;` This will short-circuit forEach because some array elements in `res` can be undefined. (This will make sense if u look at line 210 in map controller suggestion!